.eslintrc.json

```diff
@@ -160,6 +160,14 @@
       {
         "ignoreComments": true
       }
-    ]
+    ],
+
+
+    //  TODO: remove the following for TS, just temporary 08/22
```

Comment:
these actually do a really good job
they convert whatever they can from var to let/const
I didn't commit to leave it clean